test/functional/terminal/cursor_spec.lua

```diff
@@ -144,7 +144,7 @@ describe('cursor with customized highlighting', function()
       [6] = {foreground = 130},
     })
     screen:attach(false)
-    execute('term "' ..nvim_dir.. '/tty-test"')
+    execute('call termopen(["' ..nvim_dir.. '/tty-test"]) | startinsert')
```

Comment:
It is rather strange that `..nvim_dir..` is surrounded by spaces, but there are no spaces between both `..` and `nvim_dir`.